Dshop/apps/products_catalogue/views.py

```diff
@@ -47,3 +48,46 @@ def get(self, request, *args, **kwargs):
 
         xml_response = self.generate_xml_file_for_ceneo(products)
         return xml_response
+
+def GetCeneoCategories(request):
```

Comment:
since it's a function it should start with small letter
```suggestion
def getCeneoCategories(request):
```
or should be changed to class. Class here would be helpful for consistency reasons only, so function is ok.

Reply:
done